### ISSUE TITLE
NTR: MRI: tuberculoma by T1, T2 properties (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8940]
New term request MRI: tuberculoma by T1, T2 properties (samant_101122000353):   UMLS: 0085388

**Comment by pnrobinson** [comment]
@MickeySegal

this bundles too much information in one term. Probably Tuberculoma would be better. What about location?

**Comment by MickeySegal** [comment]
How about "Brain tuberculoma"?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000585
- label: Brain parenchymal tuberculoma
- parents: [Abnormal brain morphology (HP:0012443)](https://hpo.jax.org/app/browse/term/HP:0012443)
- pmid: PMID:34900500
- def: Tuberculoma results from hematogenous spread of Mycobacterium tuberculosis (M. tb) from an extracranial source. Diagnosis is based on computed tomography (CT) scan and magnetic resonance imaging (MRI) studies with a similar ring-enhancing lesion.
- comment: It is estimated that TB in the brain parenchyma develops in nearly one of 300 non-treated cases of pulmonary TB cases, and in half of the patients with disseminated TB. Imaging studies, such as CT and MRI with contrast enhancement, are the basis for diagnosis of tuberculoma. The most common image of tuberculoma is a ring-enhancing lesion due to the absence of blood supply in the caseous necrosis center within the tuberculoma. MRI is slightly superior for showing the size of brain lesions and helps to identify the solid caseous necrosis. Images of caseating tuberculoma are generally composed of three zones, an inner iso-intense and hypo-intense layer image due to caseous necrosis (signals in T1WI and T2WI, respectively). In this zone, fluid-attenuated inversion recovery (FLAIR) images reflect extensive necrosis and hypercellularity. A middle layer with hypo-intense and hyperintense signals due to the presence of Langhans giant cells, epithelioid cells, and edema (in T1WI and T2WI, respectively) is enhanced with gadolinium in contrast images, whereas the external layer shows iso-intense and hypo-intense component (signals in T1WI and T2WI) due to the collagenous capsule.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8940
- synonyms:
